.github/workflows/release-facebook.yml

```diff
@@ -0,0 +1,90 @@
+name: Release Facebook
+
+on:
+  workflow_dispatch:
+    inputs:
+      page_id:
+        description: 'The pageId part of /${pageId}/photos URL'
+        required: true
+        default: 'msenjp'
+  push:
+    branches:
+      - release
+      - 317-add-workflow-for-facebook-post
+
+jobs:
+  release_facebook:
+    runs-on: ubuntu-latest
+
+    environment:
+      name: Release
+
+    env:
+      BLOG_URL: https://mseeeen.msen.jp/rss.xml
+      RSS_FILENAME: rss.xml
```

Comment:
The extension is `.xml` but the actual content will be JSON.

Reply:
I fixed the extension to `.json`.